drill-shaded/guava-shaded/pom.xml

```diff
@@ -0,0 +1,147 @@
+<?xml version="1.0"?>
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.apache</groupId>
+    <artifactId>apache</artifactId>
+    <version>18</version>
+  </parent>
+
+  <groupId>org.apache.drill</groupId>
+  <artifactId>guava-shaded</artifactId>
+  <version>${dep.guava.version}</version>
+  <name>drill-shaded/guava-shaded</name>
+
+  <packaging>jar</packaging>
+
+  <properties>
+    <!-- skip source archives -->
+    <assembly.attach>false</assembly.attach>
+    <dep.guava.version>23.0</dep.guava.version>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
+      <version>${dep.guava.version}</version>
+      <type>jar</type>
+    </dependency>
+  </dependencies>
+
+  <build>
+
+    <sourceDirectory>${project.build.directory}/shaded-sources</sourceDirectory>
+
+    <plugins>
+      <plugin>
```

Comment:
move to the parent pom. Bind `default-compile` and `default-testCompile` to `phase` `none`. The same for other plugins.

Reply:
I mean bind to `phase` `none` execution `id` that need to be disabled. For compiler the plugin it is `default-compile`.

Reply:
Sorry, but either I don't understand your intention or you missed this change in the last commit https://github.com/apache/drill/pull/1264/commits/872afebe0fb0fba68f971aaf4243716814753c43. Could you please take a look at this commit or clarify, what should be done.

Reply:
It meaningless to bind `default-compile` or `default-testCompile` execution `id` for `maven-surefire-plugin` as it does not define such execution `id`. The intention of binding default execution `id` to phase `none` is to disable execution of those plugins when desired, instead of relying on other settings (like `skip` or `skipTests`).